gui/default/syncthing/folder/editFolderModalView.html

```diff
@@ -1,4 +1,4 @@
-<modal id="editFolder" status="default" icon="{{editingExisting ? 'fas fa-pencil-alt' : 'fas fa-folder'}}" heading="{{editingExisting ? 'Edit Folder' : 'Add Folder' | translate}}" large="yes" closeable="yes">
+<modal id="editFolder" status="default" icon="{{editingExisting ? 'fas fa-pencil-alt' : 'fas fa-folder'}}" heading="{{editingExisting ? 'Edit Folder ('+folderLabel(currentFolder.id)+')' : 'Add Folder' | translate}}" large="yes" closeable="yes">
```

Comment:
So, when there is something like `{{ 'whatever' | translate }}` that means the `'whatever'` will get looked up into whatever the local language is. We can't have that be dynamic. The folder ID needs to be added outside the translation stuff.

(And you can just push more commits to the same branch instead of creating a new PR)